client/blocks/jitm/templates/modal-style.scss

```diff
@@ -25,6 +25,8 @@ $wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
 			min-width: 90vw;
 			left: 5vw;
 			right: 5vw;
+			height: auto;
+			min-height: unset;
```

Comment:
Overriding a `min-height` defined elsewhere that is causing the modal on mobile to take up height that required, resulting in extra white space underneath the CTA.